prometheus-to-sd/translator/translator.go

```diff
@@ -77,14 +102,15 @@ func OmitComponentName(metricFamilies map[string]*dto.MetricFamily, componentNam
 	return result
 }
 
-func getStartTime(metrics map[string]*dto.MetricFamily) time.Time {
+func getStartTime(metrics map[string]*metricFamilyWithTimestamp) time.Time {
 	// For cumulative metrics we need to know process start time.
 	// If the process start time is not specified, assuming it's
 	// the unix 1 second, because Stackdriver can't handle
 	// unix zero or unix negative number.
 	startTime := time.Unix(1, 0)
-	if family, found := metrics[processStartTimeMetric]; found && family.GetType() == dto.MetricType_GAUGE && len(family.GetMetric()) == 1 {
-		startSec := family.Metric[0].Gauge.Value
+	glog.Infof("getStartTime(%v)", metrics)
```

Comment:
Do we need this?

Reply:
Nope, leftover from debugging. Thanks for spotting!